app/src/main/java/org/mifos/mobile/ui/fragments/ClientAccountsFragment.java

```diff
@@ -139,13 +139,12 @@ private void setUpViewPagerAndTabLayout() {
         viewPager.addOnPageChangeListener(new ViewPager.OnPageChangeListener() {
             @Override
             public void onPageScrolled(int position, float positionOffset,
-                    int positionOffsetPixels) {
+                                       int positionOffsetPixels) {
             }
 
             @Override
             public void onPageSelected(int position) {
                 getActivity().invalidateOptionsMenu();
-                ((HomeActivity) getActivity()).hideKeyboard(getView());
```

Comment:
Removing this line is unnecessary and also add screen recording for the changes done :)